src/test/run-pass/existential_type_const.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(existential_type)]
+#![feature(impl_trait_in_bindings)]
```

Comment:
Why is `impl_trait_in_bindings` used here?

Reply:
`#![feature(existential_type)]` implicitly depends on `#![feature(impl_trait_in_bindings)]` to work properly - specifically, [this code](https://github.com/rust-lang/rust/blob/481068a707679257e2a738b40987246e0420e787/src/librustc_typeck/check/mod.rs#L856) needs to run, so that we properly visit all defining uses of an existential type. Should I add a comment explaining that?

Reply:
A comment would be helpful indeed. :)